package.json

```
@@ -53,7 +53,10 @@
     "test:production": "cypress open --env testEnv=production",
     "test:master": "cypress open --env testEnv=master ",
     "test:branch": "cypress open --env testEnv=branch",
-    "test:local": "cypress open --env testEnv=local"
+    "test:local": "cypress open --env testEnv=local",
+    "test:local:dataflow": "cypress open --env testEnv=local:dataflow",
```

```suggestion
    "test:local": "cypress open --env testEnv=local:dataflow",
```
`package.json` can be configured to be Dataflow-specific. It has to be to handle separate version numbers, etc., so we might as well take advantage of that to configure the tests appropriately.